packages/shared/src/normalizeProp.ts

```diff
@@ -1,6 +1,35 @@
 import { isArray, isString, isObject, hyphenate } from './'
 import { isNoUnitNumericStyleProp } from './domAttrConfig'
 
+const hashedCustomProperty = /^[0-9a-f]{8}-/i;
```

Comment:
The generated hashes does not contain a "-" character at the end. So, the regex is not matching. A working regex: 

`const hashedCustomProperty = /^[0-9a-f]{8}$/i;`

Reply:
Hi @jahermosilla ,

Good catch! 🐛